pkg/vsphere/spec/spec_test.go

```diff
@@ -29,7 +29,7 @@ import (
 func URL(t *testing.T) string {
```

Comment:
I would vote for a test utils -- I was thinking of adding a datastore and network function.  Specifically provide the environment variable or an empty string for GOVC_DATASTORE and GOVC_NETWORK.